mergify_engine/queue/merge_train.py

```diff
@@ -1258,21 +1258,30 @@ async def update_summaries(
         if self.has_timed_out:
             conclusion = check_api.Conclusion.FAILURE
 
-        report = check_api.Result(
-            conclusion,
-            title=original_pull_title,
-            summary=unexpected_change_summary
+        check_api_summary = (
+            unexpected_change_summary
             + queue_summary
             + "\n"
             + checks_copy_summary
             + "\n"
-            + batch_failure_summary,
+            + batch_failure_summary
         )
+
+        report = check_api.Result(
+            conclusion,
+            title=original_pull_title,
+            summary=check_api_summary,
+        )
+
         for original_ctxt in original_ctxts:
             original_ctxt.log.info(
```

Comment:
We should also move the logging out of the loop (you can use `self.train.log` instead).